.prettierignore

```diff
@@ -0,0 +1,7 @@
+**/*.md
```

Comment:
@pierre, peux-tu mettre ce qui concerne le JS dans le dossier ui/ ?

Reply:
I thought it was not possible at first. But I managed to get it working.